src/exercises/exerciseTypes/findWordInContext/FindWordInContext.js

```diff
@@ -21,7 +22,8 @@ export default function FindWordInContext({
   setIsCorrect,
   moveToNextExercise,
   toggleShow,
-  toggleShowImproveTranslation,
+  reload,
```

Comment:
i see a lot of the "reload" & "setReload" pattern. 
what is it used for?

Reply:
To make sure the changes to the bookmark are immediately visible to the user both in the modal and on the page from which they clicked to edit. By having a state that gets updated, and where needed a `useEffect` dependent on that state, the component automatically re-renders - displaying e.g. `bookmark.to` which has now a different value.